.teamcity/DFastBETests/dfastcleanconfiguration.kt

```diff
@@ -0,0 +1,101 @@
+import jetbrains.buildServer.configs.kotlin.*
+import jetbrains.buildServer.configs.kotlin.buildFeatures.swabra
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.triggers.vcs
+
+object DFastCleanConfiguration : Template({
```

Comment:
can you rename the file to `buildtemplate.kt` and the the template here to `BuildTemplate`?